packages/backend/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "build",
-    "declaration": false
+    "declaration": false,
+    "declarationMap": false
```

Comment:
Why is it necessary to make this `false` for `backend` package?

Reply:
Aight, that's something we can improve later on if necessary.

Reply:
That's weird, I can't remember why this was done. I think it's fine to enable them